pkg/webhook/admission/nic-validator.go

```diff
@@ -6,40 +6,11 @@ import (
 	ovirtsdk "github.com/ovirt/go-ovirt"
 )
 
-const (
-	// NicInterfaceCheckID defines an ID of a NIC interface model check
-	NicInterfaceCheckID = CheckID("nic.interface")
-	// NicOnBootID defines an ID of a NIC on_boot == fales check
-	NicOnBootID = CheckID("nic.on_boot")
-	// NicPluggedID defines an ID of a NIC plugged == false check
-	NicPluggedID = CheckID("nic.plugged")
-	// NicVNicPortMirroringID defines an ID of a vnic_profile.port_mirroring == true check
-	NicVNicPortMirroringID = CheckID("nic.vnic_profile.port_mirroring")
-	// NicVNicPassThroughID defines an ID of a vnic_profile.pass_through == 'enabled' check
-	NicVNicPassThroughID = CheckID("nic.vnic_profile.pass_through")
-	// NicVNicCustomPropertiesID defines an ID of a vnic_profile.custom_properties presence check
-	NicVNicCustomPropertiesID = CheckID("nic.vnic_profile.custom_properties")
-	// NicVNicNetworkFilterID defines an ID of a vnic_profile.networ_filter presence check
-	NicVNicNetworkFilterID = CheckID("nic.vnic_profile.network_filter")
-	// NicVNicQosID defines an ID of a vnic_profile.qos presence check
-	NicVNicQosID = CheckID("nic.vnic_profile.qos")
-)
-
 // TODO: move to shared package with mapping definitions
```

Comment:
Is this TODO still valid?

Reply:
Yes - I envisioned this and storage maps to be used both here - for validation, and in reconciliation - for mapping from oVirt to kubevirt values.